### PR TITLE
CX-5282 Convert Video component styles to Sass

### PR DESCRIPTION
**Pull request by kopijunkie**
Convert Video component styles to Sass

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1079-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1079-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1079-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Video/style.scss [hunk 1]
```diff
@@ -41,16 +41,16 @@
   top: 0;
   background-repeat: no-repeat;
   background-size: contain;
-  width: 26*@unit;
-  height: 26*@unit;
+  width: 26*$unit;
+  height: 26*$unit;
 }
 
-.speak_out_loudIcon {
```
Comment (josokinas) [inline src/components/Video/style.scss:48]: You mentioned you don't want to change class names? The old class name should be all fine by the new regex set?

File: src/components/Video/style.scss [hunk 2]
```diff
@@ -107,45 +107,46 @@
...
 }
+
 .recordingIndicatorText {
-  font-size: @font-size-x-small;
+  font-size: $font-size-x-small;
   line-height: 1.45;
 }
 
-.fullScreenRoundedIcon() {
+@mixin full-screen-rounded-icon() {
```
Comment (josokinas) [inline src/components/Video/style.scss:141]: [Minor] For internal (non reusable outside of this file) mixins I'd recommend to start name with an underscore, so: `_full-screen-rounded-icon`

File: src/components/Video/style.scss (hunk 1 above)
Comment (kopijunkie) [reply src/components/Video/style.scss:48]: I'm fine with that, just thought that this one seemed to be different from all the existing classes & wasn't sure if a regex could handle this mix of styles.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1079-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**